### ISSUE TITLE
Need pre-trained kinetics weights for Pytorch

### ISSUE THREAD

**Issue by fmthoker** [yjxiong/tsn-pytorch#47]
Hello everyone,
Has anyone successfully converted kinetics pre-trained weights into Pytorch TSN?

**Comment by pedro-abreu** [comment]
I have converted the inception V3 weights to Keras, if you want them tell me.

**Comment by pedro-abreu** [comment]
I have .hdf5 files for TF/Keras if you want.

**Comment by fmthoker** [comment]
@pedro-abreu  can you share them with me. Email address fmthoker@gmail.com

**Comment by ekazakos** [comment]
@pedro-abreu Thanks a lot, but Im using PyTorch atm. For anyone that may be interested, you can download the caffe pretrained weights and convert them to pytorch weights using parse_caffe.py in 
tsn-pytorch/tf_model_zoo/bninception.

**Comment by pedro-abreu** [comment]
@ekazakos Thanks man, I might switch to Pytorch eventually :). @fmthoker https://drive.google.com/drive/folders/13TaIjogd2ReNsKXNJ55o3_qsDWB5KlkM?usp=sharing here you go.

**Comment by yjxiong** [comment]
Thanks to folks! Do you mind sharing it to other users?

**Comment by ekazakos** [comment]
@yjxiong Sure, will do soon!

**Comment by ekazakos** [comment]
Everyone, you can find the pretrained Kinetics weights for TSN that I converted to work with PyTorcg here: https://drive.google.com/drive/folders/1R4n8vSQJ0LtB1mPhhjMHEQrKj3UDQS1U?usp=sharing 

Load the weights normally with: 

state_dict = torch.load(pretrained)

**If you are using Pytorch 0.4 do:**

for k, v in state_dict.items():
    state_dict[k] = torch.squeeze(v, dim=0)

**Comment by yjxiong** [comment]
@ekazakos 

Thanks! Do you mind we share the files in the project website?

**Comment by ekazakos** [comment]
@yjxiong It is totally fine.

**Comment by AhmedGamal1496** [comment]
> Everyone, you can find the pretrained Kinetics weights for TSN that I converted to work with PyTorch here: https://drive.google.com/drive/folders/1R4n8vSQJ0LtB1mPhhjMHEQrKj3UDQS1U?usp=sharing
> 
> Load the weights normally with:
> 
> state_dict = torch.load(pretrained)
> 
> **If you are using Pytorch 0.4 do:**
> 
> for k, v in state_dict.items():
>     state_dict[k] = torch.squeeze(v, dim=0)

Hello, thanks for sharing the weights. Do you have RGB Difference pretrained weights as well?

**Comment by ekazakos** [comment]
@AhmedGamal1496 Im really sorry for the very late answer, I do not know how I missed this one. I do not have RGB Difference. And yes I've used only BN-Inception.
